robots/tank_netsink.h

```diff
@@ -17,8 +17,17 @@ class TankNetSink : public Tank
       : m_Connection(connection)
     {}
 
-    /* Motor command: send TNK command over TCP */
-    void tank(float left, float right) override
+    virtual ~TankNetSink()
+    {
+        try {
+            stopMoving();
+        } catch (...) {}
```

Comment:
What is this intended to catch?

Reply:
Potentially a broken socket.

Reply:
does the exception handler need to be so generic in that case?

Reply:
Hmm, yeah probably not. There are only two kinds of exceptions that we should need to catch.